spec/overview.html

```diff
@@ -43,6 +43,8 @@ <h1>API Overview</h1>
     <p>
       The Intl object is used to package all functionality defined in the ECMAScript 2021 Internationalization API Specification to avoid name collisions.
     </p>
+
+    <emu-note>If you take a look at the specification, you would realize that while the API includes a variety of formatters, it does not provide any parsing facilities. This is something intentional that has been discussed extensively and concluded after weighing in all the benefits and drawbacks of including said functionality. You could learn more about the discussion on the <a href="https://github.com/tc39/ecma402/issues/342">issue tracker</a>.</emu-note>
```

Comment:
> If you take a look at the specification, you would realize

this presumes that anyone who looks at the spec would realize anything in particular; this comes off as exclusionary (a la "just" or "simply"). Perhaps:
```suggestion
    <emu-note>While the API includes a variety of formatters, it does not provide any parsing facilities. This is intentional, has been discussed extensively, and concluded after weighing in all the benefits and drawbacks of including said functionality. See the discussion on the <a href="https://github.com/tc39/ecma402/issues/342">issue tracker</a>.</emu-note>
```
?